dietify-backend/models/product.model.js

```diff
@@ -0,0 +1,33 @@
+const mongoose = require("mongoose");
+const { productList } = require("./product.data");
+
+const ProductSchema = new mongoose.Schema({
+  name: String,
+  image: String,
+  desc: String,
+  price: String,
+  material: String,
+  brand: String,
+  inStock: Boolean,
+  fastDelivery: Boolean,
+  ratings: Number,
+  offer: String,
+  idealFor: String,
+  level: String,
+  color: String,
+}, { timestamps: true });
+
+const Product = mongoose.model("Product", ProductSchema);
+
+async function storeProducts() {
+  try {
```

Comment:
try to keep this type of functions in its route or controller since this will be used to save new product in DB